packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1078,31 +1106,6 @@ export class Grapher
         return url
     }
 
-    @computed get overlayTab(): GrapherTabOption | undefined {
-        return this.overlay
-    }
-
-    @computed get currentTab(): GrapherTabOption {
-        return this.overlay ? this.overlay : this.tab
-    }
-
-    set currentTab(desiredTab: GrapherTabOption) {
-        if (
-            desiredTab === GrapherTabOption.chart ||
-            desiredTab === GrapherTabOption.map ||
-            desiredTab === GrapherTabOption.table
-        ) {
-            this.tab = desiredTab
-            this.overlay = undefined
-            return
-        }
-
-        // table tab cannot be downloaded, so revert to default tab
-        if (desiredTab === GrapherTabOption.download && this.isOnTableTab)
-            this.tab = this.authorsVersion.tab ?? GrapherTabOption.chart
-        this.overlay = desiredTab
```

Comment:
Note for the reviewers: No need to carry this logic over since that download modal only offers a csv data file when accessed from the table tab